server/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -1067,6 +1077,33 @@ public void testFilterCacheStats() throws Exception {
         assertThat(response.getTotal().queryCache.getMemorySizeInBytes(), equalTo(0L));
     }
 
+    public void testBulkStats() throws Exception {
+        final String index = "test";
+        assertAcked(prepareCreate(index).setSettings(settingsBuilder().put("index.number_of_shards", 2)
+            .put("index.number_of_replicas", 1)));
+        ensureGreen();
+        final BulkRequest request1 = new BulkRequest();
+        for (int i = 0; i < 500; ++i) {
```

Comment:
I don't think we need to index 500 documents here? We could probably get away with a smaller amount like 20, so that the test doesn't take as long? Was this added to try and ensure that the total time was greater than 0?

Reply:
Yes ,  in order to routiing to two different shards and  ensure that the total time was greater than 0.  As you said , 20 documents is enough.